eemeter/models/temperature_sensitivity.py

```diff
@@ -32,6 +39,21 @@ class BaseloadHeatingCoolingModelParameterType(ParameterType):
         "cooling_slope"
     ]
 
+
+class BaseloadSeasonalHeatingCoolingModelParameterType(ParameterType):
+    # TODO: Should we have separate input and results parameter types?
```

Comment:
I was having trouble with the idea of starting values in the first place.  The framework is good for producing simulations, but it seems so much simpler to have a model with internal state, and the parameters are produced only from a fit (not all rely heavily on user/programmer-specified starting values).  Ideally this would mean that we don't have to manually track the parameters in multiple places.  It also implies that a prediction isn't generally valid before a fit, though.